sympy/functions/elementary/hyperbolic.py

```diff
@@ -1336,7 +1336,7 @@ def _eval_nseries(self, x, n, logx, cdir=0):  # asinh
 
         # Handling points lying on branch cuts (-I*oo, -I) U (I, I*oo)
         if (1 + arg0**2).is_negative:
-            ndir = arg.dir(x, cdir if cdir else 1)
+            ndir = arg.dir(x, cdir or 1)
```

Comment:
There are a lot of changes like this. It makes me wonder if the default should just be `1` rather than `0`.